.rubocop.yml

```diff
@@ -70,6 +70,12 @@ Style/AccessorGrouping:
 Style/ArrayCoercion:
   Enabled: true
 
+Style/AsciiComments:
+  Enabled: false
+
+Style/Encoding:
```

Comment:
and this one?

Reply:
@denys281  This cop determines when to use encoding and when not. Also used in config/initializers/money.rb